scaleph-ui-react/src/pages/Stdata/DataElement/components/DataElementForm.tsx

```diff
@@ -76,7 +76,7 @@ const MetaDataElementForm: React.FC<ModalFormProps<MetaDataElement>> = ({
           id: data.id,
           elementCode: data.elementCode,
           elementName: data.elementName,
-          dataType: data.dataType.value,
```

Comment:
should `data.dataType?.value`. 
when create button clicked, `DataElementForm` should be empty and `data.dataType` is undefined causing error, when edit button clicked, `DataElementForm` should show the edited value and `data.dataType` will have a value.
`data.dataType?.value` will check  whether `data.dataType` is defined and determine to access `value` field automatically.